rholang/src/test/scala/coop/rchain/rholang/interpreter/SubstituteTest.scala

```diff
@@ -177,3 +177,14 @@ class NewSubSpec extends FlatSpec with Matchers {
     )
   }
 }
+
+class EvalSubSpec extends FlatSpec with Matchers {
+  "Eval" should "remove Eval/Quote pairs." in {
+    val env: Env[Par] = Env.makeEnv(GPrivate("one"), GPrivate("zero"))
+    val target: Par   = Eval(ChanVar(BoundVar(1)))
+    val result: Par   = substitute(target)(env)
+    val expected: Par = GPrivate("one")
+
+    result should be(expected)
+  }
```

Comment:
I was expecting 

`Env.makeEnv(Quote(GPrivate("one")), Quote(GPrivate("zero"))`

Since you use `Quote(GPrivate("..."))` in your MatchTest, I'm assuming GPrivate is a process.

Reply:
Every name is a quoted process.
To make a private name, you quote a private process.